README.md

```diff
@@ -1,9 +1,9 @@
 # EnergyDataSimulationChallenge
 
 Welcome to EnergyDataSimulationChallenge!
-This project is to challenge the analysis of big energy production data.
+This project is to challenge the analysis/development of big energy production data.
```

Comment:
"This challenge project allows applicants to demonstrate their ability to analyze and develop software the makes use of big energy production data."